src_c/polygon.c

```diff
@@ -7,6 +7,40 @@
 #include <stddef.h>
 #include <math.h>
 
+static int
+set_polygon_center_coords(pgPolygonBase *polygon)
+{
+    if (!polygon) {
+        return 0;
+    }
+    double sum_x = 0;
+    double sum_y = 0;
+    Py_ssize_t i2; 
+    for (i2 = 0; i2 < polygon->verts_num * 2; i2 += 2) {
+        sum_x += polygon->vertices[i2];
+        sum_y += polygon->vertices[i2 + 1];
+    }
+    polygon->c_x = sum_x / polygon->verts_num;
+    polygon->c_y = sum_y / polygon->verts_num;
+    return 1;
+}
+
+static int
+move_polygon(pgPolygonBase *polygon, double dx, double dy)
```

Comment:
this should be `_pg_move_polygon_helper`

Reply:
Done!